src/topology/separation.lean

```diff
@@ -170,6 +216,33 @@ begin
     exact this rfl },
 end
 
+namespace separated
+
+open separated finset
+
+lemma finset_disjoint_finset_opens_of_t2 [t2_space α] :
+  ∀ (s t : finset α), disjoint s t → separated (s : set α) t :=
+begin
+  refine induction_on_union _ (λ a b hi d, (hi d.symm).symm) (λ a d, empty_right a) (λ a b ab, _) _,
+  { obtain ⟨U, V, oU, oV, aU, bV, UV⟩ := t2_separation
+      (by { rw [ne.def, ← finset.mem_singleton], exact (disjoint_singleton.mp ab.symm) }),
+    refine ⟨U, V, oU, oV, λ f hf, _, λ f hf, _, set.disjoint_iff_inter_eq_empty.mpr UV⟩;
+    rwa [finset.mem_singleton.mp hf] },
+  { intros a b c ac bc d,
+    rw [coe_union],
+    apply union_left (ac (disjoint_of_subset_left (a.subset_union_left b) d)) (bc _),
```

Comment:
```suggestion
    apply_mod_cast union_left (ac (disjoint_of_subset_left (a.subset_union_left b) d)) (bc _),
```
I'm suggesting this mostly for learning purposes, it doesn't make much difference here.